server/helpers/dates.js

```diff
@@ -0,0 +1,5 @@
+import format from 'date-fns/format';
+
+export const TODAY_WORDS = format(new Date(), 'd MMM yyyy');
+
+export const TODAY_DDMMYY = format(new Date(), 'ddMMyyyy');
```

Comment:
It is a bad idea to put a format type in the variable name. This is not advisable because the name will be incorrect if we decide to change the format tomorrow. Also, no one wants to memorize the code type when they are using the function.

This is better as a function than a constant. The date changes everyday, so it is not a constant. 

```suggestion
export const getTodaysDateShortCode = () => format(new Date(), 'ddMMyyyy');
```